tests/test_manager.py

```diff
@@ -172,6 +172,36 @@ def test_projectmanager_show(mock_project):
         assert mock_manager_projects.called
 
 
+def test_projectmanager_projects_list_args():
+    """
+    Is the group.projects.list() method called with the right arguments?
+
+    1. Make `api.groups.list` return a list with a single item (a group)
+    2. Verify the call to `group.projects.list` uses the right arguments
+    """
+    # TODO: make this test work
+    mock_api = Mock()
+    mock_list = Mock(return_value=[])
+    mock_group = Mock()
+    mock_group.projects.list = mock_list
+    mock_api.groups.list.return_value = [mock_group]
+
+    project_manager = ProjectManager(
+        group_filter='',
+        project_filter='',
+        topic_list=[],
+        uri=TEST_URI,
+        token=TEST_TOKEN,
+        insecure=False,
+    )
+    project_manager.api = mock_api
+
+    project_manager.projects()
```

Comment:
project_manager.projects() is not a method, its a generator method.
So you probably want to change `project_manager.projects()` to something like 
```
for value in project_manager.projects():
    pass
```
 or even
``` list(project_manager.projects()) ```
both options will fully evaluate the generator (run it until next(generator) raises a StopIteration exception)

with that change, the test works for me :)

Reply:
That's an awesome hint! Thank you, Dee-mee-tree! :100: :rocket: :bow: 
You are the best! :1st_place_medal: